packages/amplify-graphql-types-generator/src/loading.ts

```diff
@@ -5,6 +5,8 @@ import { buildClientSchema, Source, concatAST, parse, DocumentNode, GraphQLSchem
 import { ToolError } from './errors';
 import { extname, join, normalize, relative } from 'path';
 
+const originalPlatform = process.platform;
```

Comment:
I don't see the platform being used more than once, so I don't think we need this variable in global scope.

```suggestion
```